### PR TITLE
add navigation bar

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**
maybe we need to adapt the userstory??? 🤔

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -8,6 +9,9 @@ function Header() {
         <Typography variant="h5" component="h1" sx={{ flexGrow: 1 }}>
           Yummy Rater
         </Typography>
+        {/* <div>
+          <Navigation />
+        </div> */}
```
Comment (kai-franke) [inline components/Header.tsx:14]: As we're not going to use the navigation in the header, could you please remove it here?

File: components/Header.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { AppBar, Typography, Toolbar } from "@mui/material";
 import Login from "./Login";
+import Navigation from "./Navigation";
```
Comment (kai-franke) [inline components/Header.tsx:3]: This import statement is not necessary if we do not use the component here.

File: components/Layout.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import Header from "./Header";
+import Navigation from "./Navigation";
+import { ReactNode } from "react";
+
+interface AuthProps {
+  children: ReactNode;
+}
```
Comment (kai-franke) [inline components/Layout.tsx:7]: As with most TypeScript solutions: When I see it, it makes total sense, but I wouldn't have thought that this is the solution. 😄

File: components/Layout.tsx
Comment (kai-franke) [inline components/Layout.tsx:1]: Now that we have a layout component: Couldn't we also include the box here that generates the recurring main element on every page?
`<Box component="main" m={3}>`

File: components/Navigation.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import AddIcon from "@mui/icons-material/Add";
+import AccountBoxIcon from "@mui/icons-material/AccountBox";
+import Link from "next/link";
+
+import { useState } from "react";
+export default function Navigation() {
+  const [value, setValue] = useState(0);
+  return (
+    <Box sx={{ width: 500 }}>
```
Comment (kai-franke) [inline components/Navigation.tsx:18]: What is this `<Box>` element for? I don't see a difference when I remove it.

File: components/Navigation.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+          value={value}
+          onChange={(event, newValue) => {
+            setValue(newValue);
+          }}
+        >
+          <BottomNavigationAction
+            href="/"
+            label="Home"
+            icon={<HomeIcon />}
+            component={Link}
```
Comment (kai-franke) [inline components/Navigation.tsx:39]: Amazing that this works so easily! 🚀

File: components/Navigation.tsx (hunk 4 above)
Comment (Roland-Hufnagel) [reply components/Navigation.tsx:18]: I removed it.

File: pages/_app.tsx [hunk 6]
```diff
@@ -51,11 +52,21 @@ export default function App({
 
   return (
     <>
+      <Head>
```
Comment (kai-franke) [inline pages/_app.tsx:55]: Good idea!

**Review by kai-franke — APPROVED** [review]
Great job! 🍾

**Pull request merged**